backend/src/api/workflows.ts

```diff
@@ -338,4 +338,16 @@ router.post('/task/groupTask/:groupTaskID/complete', async (req, res) => {
   res.status(200).json(updatedGroupTask);
 });
 
+/**
+ * Mark group task as active.
+ */
+router.post('/task/groupTask/:groupTaskID/active', async (req, res) => {
```

Comment:
Do you think we should update the above endpoint from `/task/groupTask/:groupTaskID/complete` to `/task/groupTask/:groupTaskID/status` and then pass in [complete, active] in the body? So this would update the status from a single endpoint rather than having different ones for each status.